### DISCUSSION TITLE
Table Columns after update

### DISCUSSION THREAD

**Discussion by insomniacj93** [Budibase/budibase#9556] [Help]
Hi guys! After the recent update the value in my column rows turned into this.
![imagem](https://user-images.githubusercontent.com/107686004/217194913-c30bbd9c-d448-4a9e-a62f-34d3212f3f04.png)
Before the update it would display Test, the table itself it's pretty large, that i know. But is there any way to revert it?

Previous version 2.2.15

Current version 2.3.0

**Comment by m7kael** [comment]
Having the same issue if i use "Auto" value in the width container under column configuration:
![Screenshot 2023-02-08 at 13 33 31](https://user-images.githubusercontent.com/112936118/217531075-df99c8ad-6349-463b-a793-db3bb297a573.png)
